input/templates/partials/myaccount/edit-address-form-elements.hbs

```diff
@@ -0,0 +1,48 @@
+<div class="row">
+  <div class="col-sm-4">
+    <div class="title">
+      <span class="text-uppercase">{{i18n "main:form.title"}}</span><br>
+      <select id="shipping-title-select" class="checkout-dropdown" name="title{{suffix}}">
+        <option value="" selected>{{i18n "main:form.choose"}}</option>
+        {{#each form.salutations.list}}
+        <option value="{{value}}" {{#if selected}}selected{{/if}}>{{label}}</option>
+        {{/each}}
+      </select>
+    </div>
+  </div>
+</div>
+<div class="row">
+  {{> form/input-text columnWidth=6 name="firstName" labelKey="form.firstName" required=true value=form.firstName}}
+  {{> form/input-text columnWidth=6 name="lastName" labelKey="form.lastName" required=true value=form.lastName}}
+  {{> form/input-text columnWidth=12 name="streetName" labelKey="form.addressOne" required=true value=form.streetName}}
+  {{> form/input-text columnWidth=12 name="additionalStreetInfo" labelKey="form.addressTwo" required=false value=form.additionalStreetInfo}}
+  {{> form/input-text columnWidth=8 name="city" labelKey="form.city" required=true value=form.city}}
+  {{> form/input-text columnWidth=4 name="postalCode" labelKey="form.postCode" required=true value=form.postalCode}}
+  <div class="col-sm-6">
+    {{> checkout/choose-country containerClass="country" selectId="shipping-country-select" selectName="country" options=form.countries}}
+  </div>
+  {{> form/input-text columnWidth=6 name="region" labelKey="form.region" required=false value=form.region}}
+</div>
+<hr>
+<div class="row">
+  <div class="col-sm-6">
+    <div class="input-block">
+      <label for="billing-checkbox">
+        <span>
+          <input id="billing-checkbox" type="checkbox" value="true" name="billingAddress">
```

Comment:
The value that tells whether the address is selected as default billing/shipping should be coming from the Address in CTP, not hardcoded to true. It can come inside the `address` object in JSON, from a field called `defaultBilling`.